assets/resources/application/de.xml

```diff
@@ -417,13 +417,13 @@
 				<node name="summaryOn" value="Zeige Text in zwei Spalten wenn das Gerät horizontal gehalten wird"/>
 				<node name="summaryOff" value="Text immer in einer Spalte anzeigen"/>
 			</node>
-			<node name="showFootnoteToast" value="Show linked text in a popup" toBeTranslated="true">
-				<node name="never" value="Never" toBeTranslated="true"/>
-				<node name="footnotesOnly" value="For footnotes only" toBeTranslated="true"/>
-				<node name="footnotesAndSuperscripts" value="For footnotes and superscript links" toBeTranslated="true"/>
-				<node name="allInternalLinks" value="For all internal links" toBeTranslated="true"/>
+			<node name="showFootnoteToast" value="Zeige verknüpften Text in Popups">
+				<node name="never" value="Niemals"/>
+				<node name="footnotesOnly" value="Nur für Fußnoten"/>
+				<node name="footnotesAndSuperscripts" value="Für Fußnoten und hochgestellte Verknüfungen"/>
+				<node name="allInternalLinks" value="Für alle internen Verknüpfungen"/>
 			</node>
-			<node name="footnoteToastDuration" value="How long to display link popup" toBeTranslated="true"/>
+			<node name="footnoteToastDuration" value="Anzeigedauer für Popups"/>
```

Comment:
As for line 426 ("How long to display link popup" -- "Anzeigedauer für Popups") This option is for footnotes and similar links popups only, new version contains similar option for dictionary popups too.